types/markdown-escape/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-duplicate-imports": ["allow-namespace-imports"]
```

Comment:
Why is this needed?

Reply:
See https://github.com/palantir/tslint/issues/3418. Linter told me to merge these two lines which is not possible:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7e7194659ab906ddebf7168bfb3a66563052af05/types/markdown-escape/markdown-escape-tests.ts#L1-L2

Reply:
Resolved via c812d17557d09a0984988e6ac60b3a2b9fa63800